### PR TITLE
Added a command to display UIImages them on the Mac client

### PR DESCRIPTION
**Pull request by MonsieurDart**
Hi Jason,

I just added a command to transfer UIImages over JSTP and display them on the Mac client.

You can try:
.self
.image UIImage imageNamed:'Default'
.image self view generateImage
.image self redView generateImage

What do you think?

PS: the inclusion of NSData+Base64 should be done as a submodule, but as I think CocoaPods would be a better choice, I'll let you decide how to do this :-p).

PS2: this was really fun to implement! :-D

PS3: this time I forked the right repo… :-/

### CODE REVIEW

**Comment by jbrennan** [comment]
Looks awesome. I'll take a closer look over this later, but thanks for the contribution! Do you have any screenshots or things I should look for to test?

(In actuality I've left Shopify so I no longer have commit access. But you can be sure I'll be maintaining my fork at @jbrennan/superdb so I'll likely pull these changes in there)

**Comment by MonsieurDart** [comment]
And here it is: ;-)

SuperDebug-ImageTransferCommand

**Comment by MonsieurDart** [comment]
Ok about Shopify. So do you want me to do a pull request on your fork?

**Comment by jbrennan** [comment]
Sure, just re-open the PR you have on my repo :)

**Comment by MonsieurDart** [comment]
Done. :-)

Couldn't it be possible to reset the main branch of superdb to jbrennan?

**Comment by joshpc** [comment]
@jbrennan sorry for the delay, gave you full access to this repo too

**Comment by jbrennan** [comment]
@joshpc Thanks!

File: Debug Me/Debug Me/DBMEAppDelegate.m [hunk 1]
```diff
@@ -10,6 +10,30 @@
 
 #import "DBMEViewController.h"
 #import <SuperDBCore/SuperDBCore.h>
+#import <QuartzCore/QuartzCore.h>
+
+
+
+@implementation UIView (Screenshot)
```
Comment (jbrennan) [inline Debug Me/Debug Me/DBMEAppDelegate.m]: This should be added as a category to UIView in the SuperDBCore project, shouldn't it? That way, this functionality will be available to all projects, not just the demo project.

File: SuperDBCore/SuperDBCore/SuperInterpreter.m [hunk 2]
```diff
@@ -186,6 +187,52 @@ - (void)setupResponseHandlers {
...
+         
+         NSLog(@"[SERVER]: Transfering data for image: %@", input);
+         
+         if ([result isOK]) {
+             NSLog(@"FSOK: %@", [result result]);
+             
+#if TARGET_OS_IPHONE
+             id object = [result result];
+             if ([object isKindOfClass:[UIImage class]])
+             {
```
Comment (jbrennan) [inline SuperDBCore/SuperDBCore/SuperInterpreter.m]: Be careful on the style here. I know this is nit-picky, but we keep the braces on the same line of the if/else statements (`if (condition) {`, etc.)

**Comment by jbrennan** [comment]
@MonsieurDart Are you still interested in this pull request?

**Comment by MonsieurDart** [comment]
Yes, just need a bit more time to fix this pull request. :-)

Lot of work these days, sorry about that.

**Comment by jbrennan** [comment]
No worries at all, was just curious ;) Take your time.

**Comment by MonsieurDart** [comment]
Hop! Here are the few fixes you requested. And a Readme update, if you like it…

**Pull request merged**